README.md

```
@@ -413,6 +415,207 @@ LogEngine.error('Connection failed');
+
+// Configure emoji selector directly (for customization)
+EmojiSelector.configure({
+  customMappings: [...]
+});
+```
+
+### Output Format
+
+With emoji (enabled by default), the log format is:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)